src/main/java/org/opentripplanner/graph_builder/module/AddTransitModelEntitiesToGraph.java

```diff
@@ -167,19 +171,39 @@ private void addPathwayNodesToGraph(Graph graph) {
 
     private void addBoardingAreasToGraph(Graph graph) {
         for (BoardingArea boardingArea : transitService.getAllBoardingAreas()) {
-            TransitBoardingAreaVertex boardingAreaVertex = new TransitBoardingAreaVertex(graph, boardingArea);
+            TransitBoardingAreaVertex boardingAreaVertex =
+                    new TransitBoardingAreaVertex(graph, boardingArea);
             stationElementNodes.put(boardingArea, boardingAreaVertex);
             if (boardingArea.getParentStop() != null) {
+
+                var platformVertex = stationElementNodes.get(boardingArea.getParentStop());
+                boolean wheelchair = boardingArea.getWheelchairBoarding() == WheelChairBoarding.POSSIBLE;
+                var name = Optional.ofNullable(boardingArea.getName())
+                        .map(NonLocalizedString::new)
+                        .orElse(null);
+
                 new PathwayEdge(
-                    boardingAreaVertex,
-                    stationElementNodes.get(boardingArea.getParentStop()),
-                    new NonLocalizedString(boardingArea.getName())
+                        boardingAreaVertex,
+                        platformVertex,
+                        boardingArea.getId(),
+                        name,
+                        0,
+                        0,
+                        0,
+                        0,
+                        wheelchair
                 );
 
                 new PathwayEdge(
```

Comment:
Could `PathwayEdge.lowCost()` be used here? Maybe by adding the `wheelchair` flag to it?